tracing/src/lib.rs

```diff
@@ -310,6 +310,16 @@
 //! # }
 //!```
 //!
+//! Constant expressions can also be used as field names:
```

Comment:
let's maybe add something to this documentation that states the curly braces are necessary when using the value of a constant as a field name, like this:
```suggestion
//! Constant expressions can also be used as field names. Constants
//! must be enclosed in curly braces (`{}`) to indicate that the *value*
//! of the constant is to be used as the field name, rather than the 
//! constant's name. For example:
```